regression/contracts/function-calls-03-1/test-enf-f1.desc

```diff
@@ -2,10 +2,9 @@ CORE
 main.c
 --enforce-contract f1 _ --unwind 20 --unwinding-assertions
 ^file main\.c line 13 function f2: recursion is ignored on call to 'f2'$
-^Invariant check failed$
-^Condition: decorated\.get_recursive_function_warnings_count\(\) == 0$
-^Reason: Recursive functions found during inlining$
-^EXIT=(127|134|137)$
+^inlining_decoratort: recursive call to 'f2' during inlining$
+^Numeric exception : 0$
```

Comment:
I don't think we should keep sending error code to the user (unless in debug mode to help us diagnose a problem). Also, why keep it in the test description? Is there any info from the exception that help us understand this case? The error message seems sufficient to me. We also already have the signal `EXIT`, which has a meaning for debug purposes.